src/components/editor/HtmlEditor.css

```
@@ -0,0 +1,256 @@
+  font-style: italic;
+}
+
+/* Code */
+.html-editor-content pre {
+  margin: 1em 0;
+  padding: 1em;
+  background: #1f2937;
+  color: #f9fafb;
+  border-radius: 4px;
+  overflow-x: auto;
+  font-family: 'Consolas', 'Monaco', monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)